README.md

```diff
@@ -20,7 +20,7 @@ Persona is not for everyone; if your login system is too complex and relies on m
 5. Allows changing passwords.
 6. Allows recovering forgotten passwords.
 
-## What does it NOT do?
+## What does NOT it do?
```

Comment:
Didn't you type it wrong? It was correct before.

Reply:
Thanks for paying attention, but my version is the correct way. I can give you [this simple reference](https://www.englishgrammar.org/negative-questions/) for example. @kordonme

Reply:
Thanks for paying attention. It seems there are 2 ways to formulate negative questions and both the original version and mine are perfectly correct ([reference](https://www.englishgrammar.org/negative-questions/)). You are welcome to reject my PR. Thank you again @kordonme

Reply:
I don't disagree with you. It just sounds wrong. Maybe `What *DOESN'T it do?` is better then.